course-v2/layouts/partials/resources_header.html

```diff
@@ -8,8 +8,13 @@
 {{- else -}}
 <div class="mb-2">
   <h2>Download</h2>
+  <div class="alert-warning p-2 touch-device-only">
+    <strong>Warning</strong>
```

Comment:
I think "warning" is too strong a word for this situation. 

```suggestion
    <strong>Note: </strong>
```

That said, the class name `"alert-warning"` is fine.